src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -69,24 +69,22 @@ export const JazzBar: React.FC<JazzProps> = ({ venue }) => {
     seatedAtTable && venue?.id
   );
 
-  const isReactionsAudioDisabled = !venue.isReactionsMuted;
+  const isReactionsMuted = venue.isReactionsMuted ?? DEFAULT_REACTIONS_AUDIBLE;
 
   const {
-    isShown: isUserAudioOn,
+    isShown: isUserAudioMuted,
     toggle: toggleUserAudio,
     hide: disableUserAudio,
     show: enableUserAudio,
-  } = useShowHide(venue.isReactionsMuted ?? DEFAULT_REACTIONS_AUDIBLE);
+  } = useShowHide(isReactionsMuted);
 
   useEffect(() => {
-    if (venue.isReactionsMuted) {
+    if (isReactionsMuted) {
```

Comment:
This logic looks the wrong way around to me.
If the reactions are muted... enable the audio. That's backwards.
```suggestion
    if (!isReactionsMuted) {
```

Reply:
renamed those functions to make it less confusing df26c67